### DISCUSSION TITLE
Is there any way to use new reference preprocessor from control net?

### DISCUSSION THREAD

**Discussion by Nezarik-Intmax** [comfyanonymous/ComfyUI#692] [Q&A] [answered]
Is there any way to use new reference preprocessor from control net? 
I saw extensions for others preprocessors, but not for this

**Comment by MoonMoon82** [comment, marked as answer]
@Nezarik-Intmax I already opened an feature request:
https://github.com/comfyanonymous/ComfyUI/issues/661
But @comfyanonymous gave no feedback yet. Maybe we need to be patient.
